zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/metrics/CamundaExporterMetrics.java

```diff
@@ -40,6 +47,7 @@ public CamundaExporterMetrics(final MeterRegistry meterRegistry) {
             .publishPercentileHistogram()
             .register(meterRegistry);
     processInstancesArchived = meterRegistry.counter(meterName("archived.process.instances"));
+    archivingBatchSize = meterRegistry.counter(meterName("archived.batch.size"));
```

Comment:
:wrench: I think this metric name is confusing because the _batch size_ is set via config and does not increase with every run. We already have `archived.process.instances` and `archived.batch.operations` - how about we just have a single metric with a label to distinguish between process instances and batch operations?

Reply:
I was also torn back and forth. 

1. Micrometer API is sh*&
2. The archived is about DONE archived instances - this "batch size" is about the actual batch which is in process of being archived, so like archiving, as I said I was also not sure about an ideal name.

Reply:
Renamed in https://github.com/camunda/camunda/pull/30778